### DISCUSSION TITLE
After the update, the web interface does not ask for a password.

### DISCUSSION THREAD

**Discussion by vanillamage** [wg-easy/wg-easy#1312] [Q&A] [answered]
After the update, the web interface does not ask for a password. docker inspect -f "{{ .Config.Env }}" wg-easy shows that the password is set, but when you log in http://хх.хх.хх.хх:51821 / then it just lets you into the admin panel without a password. How to fix what to do? for the test, I put the same result on another silver

**Comment by vanillamage** [comment]
docker run -d \
  --name=wg-easy \
  -e LANG=de \
  -e WG_HOST=хх.хх.хх.хх \
  -e PASSWORD=хххххххххххххххххх \
  -e PORT=51821 \
  -e WG_PORT=51820 \
  -v ~/.wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --restart unless-stopped \
  ghcr.io/wg-easy/wg-easy

**Reply by eandersons** [reply, marked as answer]
This is due to badly communicated breaking changes by deprecating the environment variable `PASSWORD` and replacing it with `PASSWORD_HASH`:

> When set, requires a password when logging in to the Web UI. See How to generate an bcrypt hash.md for know how generate the hash.

**Reply by vanillamage** [reply]
it is not clear what needs to be done specifically? And what's the problem?    https://github.com/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md  ?

**Reply by eandersons** [reply]
Yes, what is the problem? What exactly is not clear?

`PASSWORD` must be replaced with `PASSWORD_HASH`. Instructions on how to generate a hashed password are on [How_to_generate_a_bcrypt_hash.md](/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md).
